### PR TITLE
Reject sources in query based on object ID

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR reject sources in query based on object ID.

### CODE REVIEW

**Comment by stefanv** [comment]
Is this a known astronomy concept? Otherwise, the feature seems quite cryptic. Perhaps update the doc to be more explicit?

**Comment by mcoughlin** [comment]
@stefanv I updated the docs a bit more.

File: skyportal/handlers/api/source.py [hunk 1]
```diff
@@ -354,6 +354,7 @@ def get_sources(
     has_spectrum=False,
     has_followup_request=False,
     sourceID=None,
+    nonsourceIDs=None,
```
Comment (guynir42) [inline skyportal/handlers/api/source.py:357]: Maybe `rejectedSourceIDs` or something like that?

**Review by guynir42 — APPROVED** [review]
Seems like a pretty useful thing.

**Comment by stefanv** [comment]
> @stefanv I updated the docs a bit more.

I understand the new version—thanks!

File: skyportal/handlers/api/source.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/source.py]: This is a much nicer name indeed.

**Pull request merged**